modules/cluster-limitations-local-zone.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc
+
```

Comment:
Don't forget to add a `:content-type:` - this should be a CONCEPT or REFERENCE, up to you.

Reply:
I'm not sure how I missed this. Thank you! Updated.